src/System.Memory/src/System/MemoryExtensions.cs

```diff
@@ -821,12 +821,6 @@ public static bool Overlaps<T>(this ReadOnlySpan<T> first, ReadOnlySpan<T> secon
         /// <exception cref="T:System.ArgumentNullException">
         ///     <paramref name = "value" /> is <see langword="null"/> .
```

Comment:
Do we throw when value is null or when comparer is null (or both)?

Reply:
only the comparer I think since the comparer could be handling searching for null values. So it is a mistake in the comment here.